src/platform_impl/apple/uikit/view.rs

```diff
@@ -34,6 +33,9 @@ pub struct WinitViewState {
     rotation_last_delta: Cell<CGFloat>,
     pinch_last_delta: Cell<CGFloat>,
     pan_last_delta: Cell<CGPoint>,
+
+    primary_finger: Cell<Option<usize>>,
```

Comment:
I'd prefer to store `FingerId` directly here.

```suggestion
    primary_finger: Cell<Option<FingerId>>,
```